src/uu/ln/src/ln.rs

```diff
@@ -87,14 +87,14 @@ impl UError for LnError {
 
 fn long_usage() -> String {
```

Comment:
Since we're working on this, it might be nice to refactor a bit more. It should just be possible to make this a `const LONG_USAGE: &str = "..."` instead of a function.

Reply:
Makes sense, it's fixed now.